doc/classes/Joint2D.xml

```diff
@@ -22,13 +22,15 @@
 			When set to [code]0[/code], the default value from [member ProjectSettings.physics/2d/solver/default_constraint_bias] is used.
 		</member>
 		<member name="disable_collision" type="bool" setter="set_exclude_nodes_from_collision" getter="get_exclude_nodes_from_collision" default="true">
-			If [code]true[/code], [member node_a] and [member node_b] can not collide.
+			If [code]true[/code], the two bodies bound together cannot collide with each other.
 		</member>
 		<member name="node_a" type="NodePath" setter="set_node_a" getter="get_node_a" default="NodePath(&quot;&quot;)">
-			The first body attached to the joint. Must derive from [PhysicsBody2D].
+			Path to the first body (A) attached to the joint. The node must inherit [PhysicsBody2D].
+			If left empty and [member node_b] is set, the body hangs as if attached to an invisible [StaticBody2D].
```

Comment:
This is currently only true for `PinJoint2D`. See the `GodotPhysicsServer2D::joint_make_*` methods (in contrast to the 3D equivalents). I think we should make the implementation consistent with 3D (so this documentation description becomes true), if it makes sense.

Reply:
Definitely. For now I can strip this detail from Joint2D but it should be an actual feature.